resources/plugin_uploader/test_data/dist/metadata_v4_34_0.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Are these files a necessary addition to the repo? And if so do they need to be generated and checked in for each tagged release?

Reply:
This is static test data (`test_data`), used by unit tests, no changes needed w.r.t. new release cuts.